content/docs/guide.md

```diff
@@ -44,7 +44,16 @@ $ pip3 install --user packitos
 ```
 
 You can also help us test the latest development snapshot by installing packit
-directly from the master branch:
+built from the master branch in Copr:
+
+```
+$ sudo dnf copr enable packit/packit-master
```

Comment:
Do you think we should mention `packit-releases` as well? (I've added those mainly because of the dogfooding.)

They can get more quickly to users than the ones from Fedora repositories.

Reply:
So many ways to install Packit! :smiley: 

My feeling is, mentioning it could create confusion.

From the average user POV having one way to install the latest release should be more that enough.

From our POV: having one "officially supported" way of providing these releases keeps things simple.